amethyst_rendy/src/camera.rs

```diff
@@ -402,6 +404,56 @@ impl Projection {
             Projection::Perspective(ref mut s) => s.as_matrix_mut(),
         }
     }
+
+    /// Transforms position from screen space to camera space
+    pub fn screen_to_world(
+        &self,
+        screen_position: Point2<f32>,
+        screen_dimensions: &ScreenDimensions,
+        camera_transform: &Transform,
+    ) -> Point3<f32> {
+        let screen_x = 2.0 * screen_position.x / screen_dimensions.width() - 1.0;
+        let screen_y = 1.0 - 2.0 * screen_position.y / screen_dimensions.height();
```

Comment:
Should this be 
```suggestion
        let screen_y = 2.0 * screen_position.y / screen_dimensions.height() - 1.0;
```
?

You also mentioned that (only?) vulkan has `Y` flipped.. Does it mean, that for other back-ends we still have to use `1.0 - 2.0 * screen_position.y / screen_dimensions.height()` conversion?

Reply:
No, because we treat everything as vulkan coordinate space, regardless of backend.